package.json

```diff
@@ -32,9 +32,9 @@
   "dependencies": {
     "@supercharge/promise-pool": "^1.7.0",
     "@textury/ardb": "^1.1.3",
-    "arbundles": "^0.2.11",
-    "arweave": "^1.10.16",
-    "arweave-stream-tx": "^1.0.2",
+    "arbundles": "^0.6.19",
+    "arweave": "1.11.4",
```

Comment:
Based on my experience with #69, I found `arweave` at `1.11.4` to raise the SubtleCrypto error (i suspect it might affect all 1.11.1 - 1.11.4). I think we need to set `arweave` to be `~1.10.23` and `smartweave` to be exactly `0.4.48`. The reason for specifying `smartweave` is because `smartweave@0.4.48` requires `arweave@1.11.4`, which is what we're trying to avoid.

Reply:
I just saw this PR https://github.com/Bundlr-Network/arbundles/pull/50. Assuming then that it was only `arbundles` that was invoking `SubtleCrypto` in `arweave`, then I'm all good. Thanks.